pkg/cqrs/sqlitecqrs/sqlc/models.go

```diff
@@ -28,13 +28,18 @@ type App struct {
 }
 
 type Event struct {
-	InternalID ulid.ULID
-	EventID    string
-	EventName  string
-	EventData  string
-	EventUser  string
-	EventV     sql.NullString
-	EventTs    time.Time
+	InternalID  ulid.ULID
+	AccountID   interface{}
+	WorkspaceID interface{}
+	Source      sql.NullString
+	SourceID    interface{}
```

Comment:
I know these are generated code, but would still be nice to have the type signature mapped correctly instead of an `any`